src/_utils/useOverflowHiddenOnDocument/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
+
+const DEFAULT_OVERFLOW_VALUE = 'visible'
+const OVERFLOW_HIDDEN_VALUE = 'hidden'
```

Comment:
Do we really need these constants as there are only used once and not exported?

Reply:
I feel using this constants make it clear which one is the default value.